rust/cymbal/src/frames/records.rs

```diff
@@ -66,7 +61,7 @@ impl ErrorTrackingStackFrame {
             serde_json::to_value(&self.contents)?,
             self.resolved,
             Uuid::now_v7(),
-            context
+            serde_json::to_value(&self.context)?
```

Comment:
This is a bug, I think - it'll write the json `null` value to the DB, rather than not writing anything at all (a postgres `NULL`). I think we want to avoid that, and instead either write an actual json structure, or `NULL`, so e.g. `where context is not null` queries still work. The if block above is how I was doing this previously.